aggsender/types/types.go

```diff
@@ -16,6 +16,8 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 )
 
+const nilStr = "nil"
```

Comment:
Nitpick, consider exporting this constant and using it instead of this [one](https://github.com/agglayer/aggkit/blob/cca60035aa19700242b316f15b433e109db6b63a/aggsender/aggsender_initial_state.go#L20), in the `aggsender` pkg.

Reply:
Fix: https://github.com/agglayer/aggkit/pull/354/commits/00fabf68972e6a8690351f09bfcadccbe0fb4e9d